From dd72fed973c2a1bc06a97af01bed225abee69fd2 Mon Sep 17 00:00:00 2001
From: Dimitar Dimitrov <dimitar@dinux.eu>
Date: Sun, 16 Oct 2016 20:40:28 +0300
Subject: [PATCH 12/25] ld: testsuite: Disable test that is broken by the _
 prefix added to symbols by PRU-GCC

LD's --defsym option should not mangle symbol names. Hence
pr13683.d is not valid from PRU's point of view.

2016-12-26  Dimitar Dimitrov <dimitar@dinux.eu>

ld/

	* testsuite/ld-gc/pr13683.d: Disable test that is
	broken by the _ prefix added to symbols by PRU-GCC.

Signed-off-by: Dimitar Dimitrov <dimitar@dinux.eu>
---
 ld/testsuite/ld-gc/pr13683.d | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ld/testsuite/ld-gc/pr13683.d b/ld/testsuite/ld-gc/pr13683.d
index b38b9d1642..703b17754a 100644
--- a/ld/testsuite/ld-gc/pr13683.d
+++ b/ld/testsuite/ld-gc/pr13683.d
@@ -2,7 +2,7 @@
 #source: dummy.s
 #ld: --gc-sections -e main --defsym foo=foo2 tmpdir/pr13683.o
 #nm: --format=bsd
-#xfail: sh64*-*-* iq2000-*-* lm32-*-* epiphany-*-* mips64vr-*-* frv-*-* m32c-*-* rl78-*-* rx-*-* sh-*-* powerpc*-*-eabivle msp430-*-*
+#xfail: sh64*-*-* iq2000-*-* lm32-*-* epiphany-*-* mips64vr-*-* frv-*-* m32c-*-* rl78-*-* rx-*-* sh-*-* powerpc*-*-eabivle msp430-*-* pru-*-*
 
 # Note - look for both "foo" and "foo2" being defined, non-zero function symbols
 
-- 
2.11.0

